src/main/java/com/fakesitewoopasdf/songr/Routes.java

```diff
@@ -48,6 +50,23 @@ public String rev (@RequestParam(required = false, defaultValue = "reverse->a->w
         m.addAttribute("word", helper.reverseString(word));
         return "cap";
     }
+    @GetMapping("/album/{id}")
+    public String getOneAlbum(@PathVariable long id, Model m){
+        System.out.println("============>"+ id);
```

Comment:
once these `System.out.println()` commands are no longer needed, you should take them out. they become dead code and can clutter up your code